subgraph/src/indexes.ts

```diff
@@ -1,37 +1,121 @@
 import { Address, BigInt, log } from "@graphprotocol/graph-ts";
 
-import { startEpochBlockTimestamp } from "./config";
-import { borrowEmissionsCurrentEpoch, initialIndex, supplyEmissionsCurrentEpoch, WAD } from "./constants";
-import { getOrInitMarket } from "./initializer";
-import { maxBN } from "./helpers";
+import { emissions } from "./distributions";
+import { getOrInitMarket, getOrInitMarketEpoch } from "./initializer";
+import { getAgeAndEpoch } from "./constants/getAgeAndEpoch";
+import { endTimestamps } from "./constants/endTimestamps";
+import { startTimestamps } from "./constants/startTimestamps";
+import { WAD } from "./constants";
+
+/**
+ * The emission between timestamp from and timestamp to is linear, and the two timestamps has
+ * to be in the same epoch
+ */
+const computeOneEpochDistribuedRewards = (
+  marketAddress: Address,
+  timestampFrom: BigInt,
+  timestampTo: BigInt,
+  totalSupply: BigInt,
+  emissionId: string
+): BigInt => {
+  const firstEpoch = getAgeAndEpoch(timestampFrom);
```

Comment:
I would rather keep the naming `from` and `to` to make the relationship with timestamps obvious